govcd/catalog.go

```diff
@@ -250,6 +250,54 @@ func (cat *Catalog) UploadOvf(ovaFileName, itemName, description string, uploadP
 	return *uploadTask, nil
 }
 
+// UploadOvfByLink uploads an ovf file to a catalog from remote URL.
+// Returns errors if any occur during upload from vCD or upload process. On upload fail client may need to
+// remove vCD catalog item which is in failed state.
```

Comment:
There are two nits in these two lines and you know them :)

Reply:
updated VCD